.github/workflows/pull-request-checks.yaml

```diff
@@ -23,10 +23,16 @@ jobs:
             - name: Composer install
               run: composer update --prefer-dist --no-progress --no-suggest --no-interaction --prefer-lowest
 
+            - name: PHPCS Fixer install
+              run: composer update --working-dir=.tools/php-cs-fixer
+
             # Linters
             - name: Run Composer linter
               run: composer validate --strict --no-check-lock
 
+            - name: Run lint PHP
+              run: .tools/php-cs-fixer/vendor/bin/php-cs-fixer fix --ansi --verbose --diff --dry-run
+
```

Comment:
Zouden dit nog eventueel als script kunnen toevoegen in composer.json.

Zie https://github.com/freshheads/FHTwigExtensionsBundle/blob/1.x/composer.json